book/05-distributed-git/sections/distributed-workflows.asc

```diff
@@ -39,15 +39,16 @@ With Git's branching model, it's possible for hundreds of developers to successf
 (((workflows, integration manager)))
 Because Git allows you to have multiple remote repositories, it's possible to have a workflow where each developer has write access to their own public repository and read access to everyone else's.
 This scenario often includes a canonical repository that represents the ``official'' project.
-To contribute to that project, you create your own public clone of the project and push your changes to it.
+Each developer has also own private repository for conducting changes in workspace then committing.
+To contribute to that project, you create your own public clone of the project and push your changes to it from your private repository.
```

Comment:
Not exactly. If you fork a public repo, your fork is public as well. The "private repo" in the diagram is the one on your computer. At this point in the book the reader is familiar with making changes in a local repo and pushing to a remote, so there's no need to spell it out. IMO the original wording is more clear.

Reply:
Cit. "At this point in the book the reader is familiar with making changes in a local repo.." that's can be well true. Will it also be clear here for book reader private repositories are being used in workflow being elaborated? The more gaps the book's reader needs mentally to fill by themselves - while on journey with this book - the more distant reader's arrival at target.

Reply:
OK, how about this then:

> Each developer has also their own private repository (on their own machine) for authoring commits.
> To contribute to that project, you would create your own public clone of the project, and push your changes to it from your private repository.